ghost/core/core/server/web/members/app.js

```diff
@@ -85,6 +85,14 @@ module.exports = function setupMembersApp() {
             return membersService.api.middleware.sendMagicLink(req, res, next);
         }
     );
+    membersApp.post(
+        '/api/verify-otc', 
+        bodyParser.json({limit: '5mb'}),
```

Comment:
Does this need to be 5mb? I would have thought the [default of 100kb](https://expressjs.com/en/resources/middleware/body-parser.html#limit) would be enough for this endpoint as it only receives a the 6-digit OTC and a reference?